src/main/java/io/jenkins/plugins/audit/CustomHTMLLayout.java

```diff
@@ -0,0 +1,431 @@
+        }
+    }
+
+    private StringBuilder appendLs(final StringBuilder sbuilder, final String s) {
+        sbuilder.append(s).append(Strings.LINE_SEPARATOR);
+        return sbuilder;
+    }
+
+    private StringBuilder append(final StringBuilder sbuilder, final String s) {
+        sbuilder.append(s);
+        return sbuilder;
+    }
+
+    /**
+     * Returns appropriate HTML headers.
+     * @return The header as a byte array.
+     */
+    @Override
+    public byte[] getHeader() {
+        final StringBuilder sbuf = new StringBuilder();
+        append(sbuf, "<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.01 Transitional//EN\" ");
+        appendLs(sbuf, "\"http://www.w3.org/TR/html4/loose.dtd\">");
+        appendLs(sbuf, "<html>");
+        appendLs(sbuf, "<head>");
+        append(sbuf, "<meta charset=\"");
+        append(sbuf, getCharset().toString());
+        appendLs(sbuf, "\"/>");
+        append(sbuf, "<title>").append(title);
+        appendLs(sbuf, "</title>");
+	appendLs(sbuf, "<link rel=\"stylesheet\" href=\"/jenkins/static/ce4bc61b/css/style.css\" type=\"text/css\" />");
+        appendLs(sbuf, "</head>");
+        appendLs(sbuf, "<body bgcolor=\"#FFFFFF\" topmargin=\"6\" leftmargin=\"6\">");
+        appendLs(sbuf, "<hr size=\"1\" noshade=\"noshade\">");
+        appendLs(sbuf, "Log session start time " + new java.util.Date() + "<br>");
+        appendLs(sbuf, "<br>");
+        appendLs(sbuf,
+                "<table cellspacing=\"0\" cellpadding=\"4\" border=\"1\" bordercolor=\"#224466\" width=\"100%\">");
+        appendLs(sbuf, "<tr>");
+        appendLs(sbuf, "<th>Time</th>");
+        appendLs(sbuf, "<th>Thread</th>");
+        appendLs(sbuf, "<th>Level</th>");
+        appendLs(sbuf, "<th>Logger</th>");
+        if (locationInfo) {
+            appendLs(sbuf, "<th>File:Line</th>");
+        }
+        appendLs(sbuf, "<th>Message</th>");
+        appendLs(sbuf, "</tr>");
+        return sbuf.toString().getBytes(getCharset());
+    }
+
+    /**
+     * Returns the appropriate HTML footers.
+     * @return the footer as a byte array.
+     */
+    @Override
+    public byte[] getFooter() {
+        final StringBuilder sbuf = new StringBuilder();
+        appendLs(sbuf, "</table>");
+        appendLs(sbuf, "<br>");
+        appendLs(sbuf, "</body></html>");
+        return getBytes(sbuf.toString());
+    }
+
+    /**
+     * Creates an HTML Layout.
+     * @param locationInfo If "true", location information will be included. The default is false.
+     * @param title The title to include in the file header. If none is specified the default title will be used.
+     * @param contentType The content type. Defaults to "text/html".
+     * @param charset The character set to use. If not specified, the default will be used.
+     * @param fontSize The font size of the text.
+     * @param font The font to use for the text.
+     * @return An HTML Layout.
+     */
+    @PluginFactory
```

Comment:
You only need one factory method (which you have below with the Builder version). This can be removed; the original version in HtmlLayout is left for backward compatibility.

Reply:
Got it. Removed.